parseAndPopulate/modulesComplicatedAlgorithms.py

```diff
@@ -47,7 +48,7 @@
 class ModulesComplicatedAlgorithms:
 
     def __init__(self, log_directory, yangcatalog_api_prefix, credentials, confd_prefix,
-                 save_file_dir, direc, all_modules, yang_models_dir, temp_dir):
+                 save_file_dir, direc, all_modules, yang_models_dir, temp_dir, ytree_dir='tests/resources/ytrees'):
```

Comment:
Do not set default value here for now. Default value already set in api/globalConfig.py or individual scripts.